packages/docs/src/content/docs/overview/configuration.md

```diff
@@ -48,6 +48,8 @@ To use a different file path:
 knip --config path/to/knip.json
 ```
 
+If you want to use a TypeScript or a JavaScript configuration file instead of JSON, then see [the dynamic configuration page](../reference/dynamic-configuration.mdx).
+
```

Comment:
Trying to keep it as light-weight as possible for newcomers early on. Can we remove this change please? Let's keep the rest.

Reply:
Dynamic configuration is a relatively advanced feature. We're linking to it from the link you've added below, I feel that's enough.

Reply:
ok

Reply:
should be removed now